src/zjs_gpio.c

```diff
@@ -325,12 +379,12 @@ static jerry_value_t zjs_gpio_open(const jerry_value_t function_obj,
         // Zephyr ISR callback init
```

Comment:
@daniel-thompson, yes it seems more sensible to just always create a handle. Also, thanks for pointing out the getter/setter issue. Yes I think it is possible and I will look into moving in that direction. Similiarly, the functions above should also be coming from a prototype object instead of duplicated for each object. I submitted the first patch making that kind of change as PR #471.

I don't think it has to hold up this patch currently though.